crates/stdarch-verify/src/lib.rs

```diff
@@ -101,7 +109,23 @@ fn functions(input: TokenStream, dirs: &[&str]) -> TokenStream {
             } else {
                 quote! { None }
             };
-            let required_const = find_required_const(&f.attrs);
+
+            let required_const = find_required_const("rustc_args_required_const", &f.attrs);
```

Comment:
Won't this ignore the ` #[rustc_legacy_const_generics]` attributes when the verifier checks the required consts, since this empty list will be passed to it [here](https://github.com/rust-lang/stdarch/blob/aba9587a3064024aa2ffc2f7d96ba576f11b267a/crates/stdarch-verify/src/lib.rs#L147) ?